src/tape/icp.jl

```diff
@@ -106,16 +106,47 @@ function reverse_pass!(tape::InstructionTape, input::AbstractArray)
         t = tape
         t[i].output.value = t[i].output.value ∩ t[i].cache
         op = IntervalContractors.reverse_operations[Symbol(t[i].func)]
+        if op == :mul_rev
+            if istracked(t[i].input[1]) && istracked(t[i].input[2])
```

Comment:
This `if` seems to just be a combination of the two below, so can be eliminated (replacing the `elseif`s by `if`s).

Reply:
As far as I can tell, there's just two things to check: `a` or `!a`,  and *separately* `b` or `!b`

Reply:
But isn't that four comparisions in the worst case, instead of 3 if `elseif` is used?